### PR TITLE
Add log analyzer

### PR DESCRIPTION
**Pull request by michimune**
Adding /loganalyzer app to the kudu container so we can analyze application errors.

### CODE REVIEW

File: 1.7/Dockerfile [hunk 1]
```diff
@@ -45,6 +45,7 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 \
   && wget -O /tmp/npm-3.10.3.zip https://github.com/npm/npm/archive/v3.10.3.zip \
   && wget -O /tmp/npm-3.10.10.zip https://github.com/npm/npm/archive/v3.10.10.zip \
   && wget -O /tmp/npm-5.0.3.zip https://github.com/npm/npm/archive/v5.0.3.zip \
+  && ln -s ../mods-available/xml2enc.load /etc/apache2/mods-enabled \
```
Comment (nickwalkmsft) [inline 1.7/Dockerfile]: Can you actually move this down as low as you can in the Dockerfile? It's nice to keep related stuff together, and that's what we have been doing up until now, but it radically increases build and push times.

File: 1.7/kudu.conf [hunk 2]
```diff
@@ -20,6 +20,16 @@ Listen 8181
             DirectoryIndex index.aspx index.html Default.cshtml
     </Directory>
 
+    Alias /loganalyzer "/opt/LogAnalyzer"
```
Comment (nickwalkmsft) [inline 1.7/kudu.conf]: Cool!

File: 1.7/Dockerfile (hunk 1 above)
Comment (nickwalkmsft) [reply 1.7/Dockerfile]: Ah never mind, the other file changes will invalidate the build cache anyway. Ignore this.

**Comment by michimune** [comment]
Yes, if I can resolve the conflict.

**Comment by nickwalkmsft** [comment]
I think it's just because I moved off of the 1.X folder structure, everything is in "kudu" now.

**Pull request merged**